src/drivers/uart/src/i8250.rs

```diff
@@ -55,26 +27,58 @@ impl<'a> I8250<'a> {
     }
 }
 
+#[allow(dead_code)]
```

Comment:
Why allow dead code?

Reply:
Allow dead code means the UART constants can be included even though a lot of them are unused. Not sure whether I should remove them.